admin/class-plugin-conflict.php

```diff
@@ -158,6 +161,9 @@ public static function hook_check_for_plugin_conflicts( $plugin = false ) {
 		/* translators: %2$s expands to 'RS Head Cleaner' plugin name of possibly conflicting plugin with regard to differentiating output between search engines and normal users. */
 		$plugin_sections['cloaking'] = __( 'The plugin %2$s changes your site\'s output and in doing that differentiates between search engines and normal users, a process that\'s called cloaking. We highly recommend that you disable it.', 'wordpress-seo' );
 
+		/* translators: %1$s expands to Yoast SEO, %2$s: 'All in One SEO Pack' plugin name of possibly conflicting plugin with regard to the creation of duplicate SEO meta. */
```

Comment:
This translator comment feels wrong. In the case of the cloaking it is a RS Head Cleaner plugin. But in this case it should be 'SEO' plugin name instead of 'All in One SEO Pack'.